src/platform/android/java/chip/platform/DefaultConfigurationManager.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *   Copyright (c) 2021 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package chip.platform;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+import java.util.Base64;
+import java.util.Map;
+
+/** Java interface for ConfigurationManager */
+public class DefaultConfigurationManager implements ConfigurationManager {
```

Comment:
Class name should describe what the class does, not how it's used, so rename `PreferencesConfigurationManager`?

Reply:
fixed at 21c564d